src/Rule/EnumMulti.php

```diff
@@ -53,7 +53,7 @@ public function setParameter($params)
 	public function getMessageParameters()
 	{
 		return array(
-			'values' => implode('|', $this->getParameter()),
+			'value' => implode('|', $this->getParameter()),
```

Comment:
No, it's not ok to change this. It will break custom error messages that someone might have set.